python/ray/autoscaler/_private/kubernetes/config.py

```diff
@@ -105,6 +116,9 @@ def get_autodetected_resources(container_data):
         if node_type_resources[key] == 0:
             del node_type_resources[key]
 
+    memory_limits = _get_resource(container_resources, "memory", "limits")
+    node_type_resources["memory"] = int(memory_limits * 0.6)
```

Comment:
should this be `memory_total * (1 - ray_constants.DEFAULT_OBJECT_STORE_MEMORY_PROPORTION)`?